test/test_helper.rb

```diff
@@ -37,6 +37,23 @@ def assert_logs_match(regexp, times = nil)
         assert_match regexp, @logger_stream.read
       end
     end
+
+    alias_method :orig_assert_raises, :assert_raises
+    def assert_raises(*args)
+      case args.last
```

Comment:
I see the value of the override but we should keep it mind that we could no longer use the last argument feature (http://www.rubydoc.info/gems/minitest/5.10.1/Minitest/Assertions#assert_raises-instance_method):

```ruby
assert_raises(WhateverError, "does not raise when logging is disabled") { ... }
```